ServerCore/Pages/EventSpecific/FAQ.cshtml

```diff
@@ -38,6 +38,14 @@ else
             if ((iframe !== null) && (iframe !== undefined)) {
                 iframe.contentWindow.postMessage("", "*");
             }
+            setTimeout(() => {
+                const iframe = document.querySelector("#faq-frame");
+                if ((iframe !== null) && (iframe !== undefined)) {
+                    if (iframe.classList.contains("unsized")) {
+                        iframe.style.display = "none";
```

Comment:
Can we also display something explaining what likely happened and how to fix it? This will be very difficult to debug when something is wrong, especially if you are not a member of the tech team.

Reply:
The old partials only displayed the message for admins and authors.  If we're okay with the message also displaying for players, then sure.